pkg/e2e/http_test.go

```diff
@@ -51,7 +51,7 @@ func TestHTTPEndpointCall(t *testing.T) {
 				return errors.New("not ready")
 			}
 
-			t.Errorf("Unexpected status code: %s. Resp: %v", status, body)
+			t.Errorf("Unexpected status code: %d. Resp: %v", status, body)
```

Comment:
why not just `%v`? much harder to make mistakes with that

Reply:
Not sure what is the style guide here but I've changed it to %v.